twoliter/src/main.rs

```diff
@@ -1,3 +1,5 @@
+extern crate core;
+
 use crate::cmd::{init_logger, Args};
```

Comment:
I think `extern crate` is deprecated? Did your IDE add this? 

```suggestion
use core;
use crate::cmd::{init_logger, Args};
```

Reply:
Thats weird